source/java/src/org/lucee/extension/orm/hibernate/util/CommonUtil.java

```diff
@@ -0,0 +1,1068 @@
+package org.lucee.extension.orm.hibernate.util;
```

Comment:
hard to see the change, when the function not got moved but deleted and created. did you just chnage the package name or also code inside the class. best do not move stuff around at all,this can be done is a separate "clean up PR". 
In short move the class back to it's old package